include/common/test.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef TEST_H
+#define TEST_H
+
+#ifdef TEST
+int test_crust_boots(void);
```

Comment:
`run_tests` goes here, not this function

Reply:
done